guides/getting-started/getting-started-openshift.adoc

```diff
@@ -46,8 +46,8 @@ To spin up a Keycloak server in your project, execute the following command:
 [source,bash,subs="attributes+"]
 ----
 oc process -f https://raw.githubusercontent.com/keycloak/keycloak-quickstarts/latest/openshift-examples/keycloak.yaml \
-    -p KEYCLOAK_USER=admin \
-    -p KEYCLOAK_PASSWORD=admin \
+    -p KEYCLOAK_ADMIN=admin \
+    -p KEYCLOAK_ADMIN_PASSWORD=admin \
```

Comment:
AFAIK, we're not using H2 by default, so this won't work. DB has to be explicitly set.

Reply:
sync with comments on:
https://github.com/keycloak/keycloak-quickstarts/pull/304

Reply:
don and it doesn't require changes in this PR